.gitignore

```diff
@@ -1,3 +1,5 @@
+.DS_Store
+.idea/

```

Comment:
Fair enough, just figured I'd mention it seeing as in open-source projects, everyone and their mother have different opinions on which IDE/Editor they prefer. Just keeps things cleaner when those exclusions are user managed rather than repository managed.